Rakefile

```diff
@@ -352,6 +353,9 @@ task :cross do
 end
 
 desc "build a windows gem without all the ceremony."
-task "gem:windows" => %w[cross native gem]
+task "gem:windows" do
+  require "rake_compiler_dock"
+  RakeCompilerDock.sh "bundle && rake cross native gem MAKE='nice make -j`nproc`' RUBY_CC_VERSION=#{ENV['RUBY_CC_VERSION']}"
```

Comment:
@knu Thanks for trying out! Your Gemfile.lock is seems to be rather old. You could update it, or add `RUBYOPT=--disable-rubygems` to the rake call above.